modules/search/class.jetpack-search-template-tags.php

```diff
@@ -135,6 +153,89 @@ public static function render_filter( $filter, $default_post_types ) {
 		<?php
 	}
 
+	/**
+	 * Renders a single filter for instant search.
+	 *
+	 * @since 8.3.0
+	 *
+	 * @param array $filter             The filter to render.
+	 */
+	public static function render_instant_filter( $filter ) {
+		if ( empty( $filter ) || empty( $filter['buckets'] ) ) {
+			return;
+		}
+
+		$data_base = '';
+		$qv        = $filter['buckets'][0]['query_vars'];
+		$tax_key   = '';
+		switch ( $filter['buckets'][0]['type'] ) {
+			case 'taxonomy':
+				$data_base = 'data-filter-type="' . $filter['buckets'][0]['type'] . '" ';
+				foreach ( $qv as $k => $v ) {
```

Comment:
ideally should confirm $qv is iterable with foreach : null, etc this will generate warnings.

Reply:
Apologies not sure why foreach is in use here. foreach will lead to $tax_key being the last item? Is it possible to have multiple query_vars in the bucket? Was this approach simply to get the singular unknown key?

Reply:
cc @gibrown

Reply:
Ugh, ya nice catch. This worked but is dumb and was related to an earlier version of the code.